reporting/miro_transformer/src/transform.py

```diff
@@ -0,0 +1,83 @@
+import json
+from datetime import datetime
+
+
+def transform(record):
```

Comment:
minor: a good name for this variable would be `miro_transformable`, as the type we use in the catalogue transformer is `MiroTransformable`.